chore: add more docs (#1130)
netlify/netlify-plugin-nextjs: README.md

package = "@netlify/plugin-nextjs"
```

## Deploying

If you build on Netlify, this plugin will work with no additional configuration. However if you are building and

> deploying locally using the Netlify CLI, you need to ensure that you deploy using `netlify deploy --build`. Running the

Reviewer: [sand] A suggestion to make this wording stronger.
```suggestion
deploying locally using the Netlify CLI, you must deploy using `netlify deploy --build`. Running the
```

support. See [`demos/next-export`](https://github.com/netlify/netlify-plugin-nextjs/tree/main/demos/next-export) for an
example.

## Asset optimization

Netlify [asset optimization](https://docs.netlify.com/site-deploys/post-processing/) should not be used with Next.js
sites. Assets are already optimized by Next.js at build time, and doing further optimization can break your site. Ensure

> that it is not enabled at **Site settings > Build & deploy > Post processing**.

Reviewer: [sand] Just adding the final section in the UI to this path.
```suggestion
that it is not enabled at **Site settings > Build & deploy > Post processing > Asset optimization**.
```
